core/src/test/java/com/linecorp/armeria/server/ServerBuilderTest.java

```diff
@@ -64,4 +67,106 @@ public void numMaxConnections() {
         final ServerBuilder sb = new ServerBuilder();
         assertThat(sb.maxNumConnections()).isEqualTo(Integer.MAX_VALUE);
     }
+
+    /**
+     * Makes sure each virtual host can have its custom logger name.
+     */
+    @Test
+    public void setAccessLoggerTest1() {
+        final Server sb = new ServerBuilder()
+                .http(0)
+                .service("/", (ctx, req) -> HttpResponse.of(HttpStatus.OK))
+                .accessLogger(
+                        LoggerFactory.getLogger("default"))
+                .withVirtualHost("*.example.com")
+                .and()
+                .withVirtualHost("*.example2.com")
+                .accessLogger("com.ex2")
+                .and()
+                .withVirtualHost("*.example3.com")
+                .accessLogger((host) -> LoggerFactory.getLogger("com.ex3"))
+                .and()
+                .virtualHost(new VirtualHostBuilder("*.example4.com", "def.example4.com").build())
+                .virtualHost(new VirtualHostBuilder("*.example5.com", "def.example5.com")
+                                     .accessLogger("com.ex5").build())
+                .build();
+        assertThat(sb.config().defaultVirtualHost()).isNotNull();
+        assertThat(sb.config().defaultVirtualHost().accessLogger().getName()).isEqualTo("default");
+
+        assertThat(sb.config().findVirtualHost("*.example.com").accessLogger().getName())
+                .isEqualTo("default");
+
+        assertThat(sb.config().findVirtualHost("*.example2.com").accessLogger().getName())
+                .isEqualTo("com.ex2");
+
+        assertThat(sb.config().findVirtualHost("*.example3.com").accessLogger().getName())
+                .isEqualTo("com.ex3");
+
+        assertThat(sb.config().findVirtualHost("*.example4.com").accessLogger().getName())
+                .isEqualTo("default");
+
+        assertThat(sb.config().findVirtualHost("*.example5.com").accessLogger().getName())
+                .isEqualTo("com.ex5");
+    }
+
+    @Test
+    public void setAccessLoggerTest2() {
+        final Server sb = new ServerBuilder()
+                .http(0)
```

Comment:
IIRC. this is redundant. Armeria automatically adds a port if a user did not specify a port.